brave/brave-core
# Untie from Android System Auto Sync settings

Resolves https://github.com/brave/brave-browser/issues/34136



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Setup Brave Sync on Android
2. Exit browser
3. Turn off system Auto Sync settings, see [Why won’t my data Sync on my Android device?](https://support.brave.com/hc/en-us/articles/360047642371-Sync-FAQ)  
4. Launch Brave Browser
5. Expected not to see the informer as on screenshots from https://github.com/brave/brave-core/pull/7316 and have green statuses at brave://sync-internals
